pyte/streams.py

```diff
@@ -259,18 +273,18 @@ def create_dispatcher(mapping):
                 #    recognizes ``ESC % C`` sequences for selecting control
                 #    character set. However, in the current version these
                 #    are noop.
-                char = yield
+                char = yield None
```

Comment:
I understand why you did that, but I think this function is complex enough and `yield None` adds unnecessary visual noise. Can you revert unless there are good reasons to keep `yield None`?

Reply:
mypy doesn't like bare `yield`

Reply:
Would you rather I `# type: ignore` these lines?

Reply:
Example without `None`:
```
pyte/streams.py:276: error: Yield value expected  [misc]
Found 1 error in 1 file (checked 9 source files)
```

Reply:
Thanks for the context. Let's keep the `None` then. Although, it does seem a bit odd that mypy rejects syntactically valid Python.